src/handler.rs

```diff
@@ -83,6 +85,11 @@ impl Handler {
             Box::new(DocumentSymbolHandler::default()),
         );
 
+        mapping.insert(
+            "textDocument/completion".to_string(),
```

Comment:
Could we add a new handler test for this?

Reply:
yeah, I was just holding off till I could get some feed back that everything is the way we want it to work.